src/tracing_tool_output.ml

```diff
@@ -110,8 +101,35 @@ module Serve = struct
   ;;
 end
 
+module Share = struct
+  type t = { share_command_filename : string }
+
+  let maybe_param =
+    Option.map Env_vars.share_command_filename ~f:(fun share_command_filename ->
+        let%map_open.Command share = flag "share" no_arg ~doc:"Share trace." in
```

Comment:
That help text could say more. If you don't want it to be too long, I'm happy with the docs just being `magic-trace.org/w/share` and then we explain how sharing works on the wiki somewhere.

Serve could arguably work the same way.